ci/travis/format.sh

```diff
@@ -5,9 +5,18 @@
 # Cause the script to exit if a single command fails
 set -eo pipefail
 
+FLAKE8_VERSION_REQUIRED="3.7.7"
+YAPF_VERSION_REQUIRED="0.23.0"
+
+YAPF_DOWNLOAD_COMMAND_MSG="Run pip install yapf==$YAPF_VERSION_REQUIRED Make sure you are using python 3 interpreter"
```

Comment:
```suggestion
YAPF_DOWNLOAD_COMMAND_MSG="pip install yapf==$YAPF_VERSION_REQUIRED"
```

no need for the Python 3 part, we only support python 3 now :)

Reply:
Fixed!